CMakeLists.txt

```diff
@@ -376,11 +376,25 @@ if(WITH_NETWORK_BACKEND)
 		endif()
 	endif()
 
-	target_sources(iio PRIVATE network.c)
```

Comment:
Commint description wasn't changes. Is the one for usb.

Reply:
Looks like a bad copy/paste in the commit message.

Reply:
Updated.